christfellowshipchurch/web-app-v2
# External Web Updates

### About
This PR closes 4 tickets 

1. [CFDP-2948](https://christfellowshipchurch.atlassian.net/jira/software/projects/CFDP/boards/2?selectedIssue=CFDP-2948) - Include "40 years"
2. [CFDP-2946](https://christfellowshipchurch.atlassian.net/jira/software/projects/CFDP/boards/2?selectedIssue=CFDP-2946) - Newsletter Subscribe
3. [CFDP-2929](https://christfellowshipchurch.atlassian.net/jira/software/projects/CFDP/boards/2?selectedIssue=CFDP-2929) - Get There First subtitle update
4. [CFDP-2914](https://christfellowshipchurch.atlassian.net/jira/software/projects/CFDP/boards/2?selectedIssue=CFDP-2914) - Espanol to menus and footer

### Test Instructions
1. Make sure that under under the Thrive in Every Area… section it now says "40 years", not "35 years"
2. The Subscribe button all the way at the bottom of the page now routes to [this page](https://rock.christfellowship.church/page/4344)
3. Under "A church for every generation." the subtitle now is "In the race to the heart of the next generation, the first one there wins. [Learn about our vision](http://localhost:3000/get-there-first)
to Get There First."
4. You should now see Iglesia Español under web and mobile menu and footer, and all of them should link to `/discover?c=en-espanol&s=`

### Screenshots
<img width="617" alt="Screen Shot 2024-03-05 at 9 38 56 PM" src="https://github.com/christfellowshipchurch/web-app-v2/assets/46769629/279fe1d6-ff32-406d-96e4-93c7eeaa83fe">

<img width="1069" alt="Screen Shot 2024-03-05 at 9 39 27 PM" src="https://github.com/christfellowshipchurch/web-app-v2/assets/46769629/ff5f8c82-dd19-4b96-a33e-0abc3c6f0917">


### **Espanol** 
<img width="344" alt="Screen Shot 2024-03-06 at 1 53 47 PM" src="https://github.com/christfellowshipchurch/web-app-v2/assets/46769629/ca605f31-829b-467d-8de6-9d452fe64bcb">

<img width="575" alt="Screen Shot 2024-03-06 at 1 53 58 PM" src="https://github.com/christfellowshipchurch/web-app-v2/assets/46769629/4c26a955-4a0c-48c4-aa2d-afbfe186e45c">

<img width="353" alt="Screen Shot 2024-03-06 at 1 54 13 PM" src="https://github.com/christfellowshipchurch/web-app-v2/assets/46769629/fc7baf83-6f50-4861-b0fb-a66f9d54c47c">

[CFDP-2948]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2948?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CFDP-2946]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2946?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CFDP-2929]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2929?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[CFDP-2914]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2914?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
